packages/component/src/BasicTranscript.tsx

```diff
@@ -155,15 +142,20 @@ const InternalTranscript = forwardRef<HTMLDivElement, InternalTranscriptProps>(
       [ref, rootElementRef]
     );
 
+    const createAvatarRendererMemoized = useMemoized(
+      (activity: WebChatActivity) => createAvatarRenderer({ activity }),
+      [createAvatarRenderer]
+    );
+
     // Flatten the tree back into an array with information related to rendering.
     const renderingElements = useMemo(() => {
-      const renderingElements: RenderingElement[] = [];
+      const renderingElements: Array<ReactNode> = [];
```

Comment:
Should we use...

```suggestion
      const renderingElements: ReactNode[] = [];
```

I am always confused when to use `Array<T>` vs. `T[]`.